### ISSUE TITLE
A licence is not specified

### ISSUE THREAD

**Issue by pikoscielniak** [mschwarzmueller/angular-2-beta-boilerplate#1]
Hi, 
I'm glad that you've made this boilerplate. I would like to use it but I don't know what its licence is. Could you specified a licence for this repo?  I'm impressed what you did on your youtube channel. Your tutorial about Angular2 is excellent for me. I can't wait for next part. Could you also reveal more social channels where I could follow your work. I couldn't find your twitter account nor email. Great job. Keep going.

**Comment by mschwarzmueller** [comment]
Hi, thanks for bringing this up. I added a license file (MIT).

Also thank you very much for the kind words - I try to offer highest quality with my YouTube channel. I only recently startet doing such videos, that's why my social network (and YouTube) presence still needs "some improvement" ;-). But I got a lot of plans in the pipeline - you're welcome to follow me on Facebook (https://www.facebook.com/mindspacechannel/) or Twitter (https://twitter.com/maxedapps).
